signing_service/signing_service/tests/test_signing_service_methods.py

```diff
@@ -417,3 +417,27 @@ def test_that_signing_service_update_daily_transactions_limit_file_name_overwrit
         self.signing_service._update_daily_transactions_limit_file_name()
         self.assertEqual(self.signing_service.daily_transactions_limit_file_name, datetime.datetime.now().strftime('%Y-%m-%d'))
         self.assertEqual(self.signing_service.signing_service_daily_transaction_sum_so_far, 0)
+
+
+class SigningServiceSetSigtermTestCase(TestCase):
+
+    def setUp(self):
+        self.host = '127.0.0.1'
+        self.port = 8000
+        self.initial_reconnect_delay = 2
+
+        with mock.patch('signing_service.signing_service.SigningService.run'):
+            self.signing_service = SigningService(
+                self.host,
+                self.port,
+                self.initial_reconnect_delay,
+                CONCENT_PUBLIC_KEY,
+                SIGNING_SERVICE_PRIVATE_KEY,
+                TEST_ETHEREUM_PRIVATE_KEY,
+                SIGNING_SERVICE_DEFAULT_RECONNECT_ATTEMPTS,
+                ConsoleNotifier(),
+            )
+
+    def test_that_set_was_sigterm_caught(self):
+        self.signing_service._set_was_sigterm_caught_true(None, None)
```

Comment:
this method's signature is `(int, Optional[Frame])` - why is it called with `(None, None)`?

Reply:
Because they are not used inside method in any way, and the test is testing something different, they don't need to be any real values. In runtime this method is called by standard lib code, which pass those values.

Reply:
I would be for using real values - it helps to understand what is really being tested and documents the code better.

Reply:
What are real values here? We don't define them in our code, they come from standard library code. I am not even sure how to create proper `FrameType` object, because I don't have to do it - standard library does it.

The documentation for SIGTERM is not something we created, it is a part of standard lib https://docs.python.org/3/library/signal.html.

Reply:
In such cases - please use `Mock()` instances or `sentinel` values - these are used to indicate that non real data is being used...

Reply:
:+1: done, please let me know if it should look like this.